minimal-notebook/Dockerfile

```diff
@@ -95,5 +95,16 @@ COPY start-notebook.sh /usr/local/bin/
 COPY jupyter_notebook_config.py /home/$NB_USER/.jupyter/
 RUN chown -R $NB_USER:users /home/$NB_USER/.jupyter
 
+# Fetch additional scripts that run Notebook server in single-user mode (for
+# use with JupyterHub)
+RUN wget \
+    -q https://raw.githubusercontent.com/jupyter/jupyterhub/master/scripts/jupyterhub-singleuser \
```

Comment:
If you point to master, there's no way to know exactly what version is getting put into a particular docker-stacks build. Nor is there a way for us to update via PR. We just have to manually force it. Suggest pointing to a stable tag (preferred) or a git SHA.